debian/bionic/foreman/control

```diff
@@ -239,3 +239,14 @@ Description: metapackage providing journald dependencies for Foreman
  a flexible systems management web application.
  .
  The foreman package should be installed along with this package.
+
+Package: foreman-service
+Architecture: all
+Section: web
+Priority: extra
+Depends: foreman (= ${binary:Version}), ${misc:Depends}
```

Comment:
As discussed above, the dependency probably does not need to be version strict here.